tests/unit/test_decorator.py

```diff
@@ -33,6 +33,10 @@
 
 
 class MyProcessor(Processor):
+    @classmethod
+    def get_default_error_builder(cls) -> "ErrorBuilderInterface":
+        return MarshmallowDefaultErrorBuilder()
+
     def validate_error(self, data_to_validate: typing.Any) -> typing.Any:
```

Comment:
the method `validate_error` has no meaning

Reply:
removed